mantaray/views.py

```diff
@@ -155,6 +156,12 @@ def add_notification(self, popup_text: str) -> None:
         self.notification_count += 1
         self._update_view_selector()
         self.irc_widget.event_generate("<<NotificationCountChanged>>")
+        if self.server_view.audio_notification:
+            try:
+                playsound("mantaray/audio/notify.mp3", False)
+            except:
+                pass
```

Comment:
- We don't want to catch literally all errors, e.g. `KeyboardInterrupt` which would make the program do nothing if you Ctrl+C while playing a sound.
- It would be good to print the full error message when an error occurs, so that you can debug why sound isn't playing.

```suggestion
            except Exception:
                traceback.print_exc()
```